File: buddybridge-backend/src/main/java/br/com/buddybridge/core/controleacesso/model/AcessoDTO.java
package br.com.buddybridge.core.controleacesso.model;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@NoArgsConstructor
@AllArgsConstructor
@Getter
@Setter
public class AcessoDTO {

    private Long idAcesso;
    private String moduloAcesso;
    private String telaAcesso;
    private String descricaoAcesso;
    private boolean ativoAcesso;

}

File: buddybridge-frontend/src/app/restrict/base/grupo_acesso/model/acessoDTO.ts
export class AcessoDTO  {
  idAcesso: number;
  moduloAcesso: string;
  telaAcesso: string;
  descricaoAcesso: string;
  ativoAcesso: boolean;

  constructor() {
    this.idAcesso = 0;
    this.moduloAcesso = '';
    this.telaAcesso = '';
    this.descricaoAcesso = '';
    this.ativoAcesso = true;
  }
}
